lib/src/cancelable.dart

```diff
@@ -5,15 +5,15 @@ part of '../mapbox_maps_flutter.dart';
 /// This can be used to manage operations that may need to be
 /// canceled before completion, such as asynchronous tasks or
 /// long-running processes.
-class AnyCancelable {
+class Cancelable {
   final void Function() cancel;
```

Comment:
Using this is a bit weird since the autocomplete suggest just plain property `myCancellable.cancel`, so I have to          append `()` myself.
I think a public `cancel()` function would be more familiar and ergonomic. In this case autocomplete would suggests exactly what user intended.

Reply:
ah you're right, I've updated to have public cancel() function :)